core/runtime.rs

```diff
@@ -1806,6 +1806,25 @@ impl JsRuntime {
   }
 }
 
+pub(crate) fn script_origin<'a>(
```

Comment:
Technically this is breaking change - it was in `bindings.rs` and it was `pub` - I suggest to revert this change, it's just moving code around.